### ISSUE TITLE
E3001 Invalid or unsupported Type AWS::SSM::Parameter in us-gov-west-1

### ISSUE THREAD

**Issue by SNieroda** [aws-cloudformation/cfn-python-lint#1384]
*cfn-lint version: cfn-lint 0.28.2*

*Description of issue: *
Error `E3001 Invalid or unsupported Type AWS::SSM::Parameter in us-gov-west-1 / us-gov-east-1` when running cfn-lint on a template in govcloud regions.

Please provide as much information as possible:
* Template linting issues:
  * Please provide a CloudFormation sample that generated the issue.
```
AWSTemplateFormatVersion: 2010-09-09
Description: IAM Roles needed for deployments

Resources:
  MyRoleParameter:
    Type: 'AWS::SSM::Parameter'
    Properties:
        Name: some-ssm-parameter
        Type: String
        Value: some value
        Description: Testing AWS::SSM::Parameter in GovCloud
```
  * If present, please add links to the (official) documentation for clarification.
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-parameter.html
  * Validate if the issue still exists with the latest version of `cfn-lint` and/or the latest Spec files
Issue exists after update, it is missing from specs for GovCloud regions, but confirmed working with the sample above.

**Comment by PatMyron** [comment]
Got someone to publish [`AWS::SSM::Parameter`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-parameter.html) to the [`us-gov-east-1`](https://s3.us-gov-east-1.amazonaws.com/cfn-resource-specifications-us-gov-east-1-prod/latest/CloudFormationResourceSpecification.json) and [`us-gov-west-1`](https://s3.us-gov-west-1.amazonaws.com/cfn-resource-specifications-us-gov-west-1-prod/latest/CloudFormationResourceSpecification.json) CloudFormation Resource Specifications. Should work now after running:

```shell
cfn-lint --update-specs
```
